pop-api/src/v0/assets/fungibles.rs

```diff
@@ -485,13 +485,11 @@ mod tests {
 	use super::FungiblesError;
 	use crate::{
 		constants::{ASSETS, BALANCES},
-		primitives::error::{
```

Comment:
Sure this works? I think this should only be changed when these types are removed from primitives crate

Reply:
Apologies, this shouldn't be changed. Just revert the changes.